modules/admin_manual/pages/installation/quick_guides/ubuntu_20_04.adoc

```diff
@@ -1,57 +1,59 @@
 = Install ownCloud on Ubuntu 20.04
 :toc: right
-:logrotate-url: https://linux.die.net/man/8/logrotate
-:webserver-user: www-data
-:webserver-group: www-data
-:install-directory: /var/www/owncloud
-:page-aliases: installation/quick_guides/ubuntu_18_04.adoc
```

Comment:
^ Not sure if it is wanted to remove all of these?

Reply:
Those are variable values which are called later in the document. But with no variables being in the document anymore - you don't need the variables being set at the top.

Last line was just weird to me. This is a 20.04 guild, why is it a relay for a 18.04 document? No one who was looking for 18.04 is going to be happy to find a 20.04 document.